README.md

```diff
@@ -146,40 +158,38 @@ The default `timeout` is 60 seconds. Minimal `timeout` is 30 seconds.
 ### Delete a Message from a Queue
 
 ```ruby
-msg.delete
+msg.delete! # => {"msg" => "Deleted"}
```

Comment:
it changes receiver, can raise error and removes message from the server. pretty "dangerous" operation.